src/components/base-address/factories/validation-factory.ts

```diff
@@ -1,6 +1,6 @@
 import { ref } from 'vue-demi'
 
-import { ValidationRule } from '@/components/base-address/enums'
+import { ValidationRule } from '@/enums/address-validation-rules'
```

Comment:
This should be `@bcrs-shared-components/enums` otherwise here (ie, when Storybook uses this component), it resolves to the local enums folder, but when used in an app, it will resolve to the app's enums folder.

Reply:
This is a problem here: https://github.com/bcgov/bcrs-shared-components/blob/4c62c41230422c5ceef744176fdc1bb04d6b5372/src/components/date-picker/DatePicker.vue#L84

But Leo said he got it working: https://github.com/bcgov/bcrs-shared-components/pull/215#discussion_r1417854456

Reply:
Fixed, thanks!